draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/color";
+
+$avatar-sm: 1.25rem;
+$avatar-md: 3rem;
+$avatar-lg: 4.5rem;
+
+.AvatarGroup {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.AvatarGroupItem,
+.AvatarCounter {
+  .small & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+
+  .small & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-sm} / -2);
+  }
+
+  .medium & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-md} / -2);
+  }
+
+  .large & [dir="rtl"] & {
+    margin-left: calc(#{$avatar-lg} / -2);
+  }
+}
+
+.AvatarGroupItem {
+  &:first-of-type {
+    margin-left: 0;
+
+    .small & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-sm} / -2);
+      }
+    }
+
+    .medium & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-md} / -2);
+      }
+    }
+
+    .large & {
+      [dir="rtl"] & {
+        margin-left: calc(#{$avatar-lg} / -2);
+      }
+    }
+  }
+}
+
+.AvatarCounter {
+  align-items: center;
+  background: $color-gray-300;
+  border: 3px solid $color-white;
+  border-radius: 100%;
+  box-sizing: border-box;
+  box-shadow: $shadow-small-box-shadow;
+  display: flex;
+  font-family: $typography-heading-1-font-family;
```

Comment:
This works fine for now, but for future proofing I would recommend specifying this for each size with the correct heading below instead, so that if we end up changing the heading-1 font, it doesn't effect the headings we've used here.

Reply:
fair call, will update that to ensure each one is declared separately 👍